types/jest/index.d.ts

```diff
@@ -801,6 +802,164 @@ declare namespace jest {
+        testEnvironment: string;
+        testEnvironmentOptions: object;
+        testFailureExitCode: string | number;
+        testLocationInResults: boolean;
+        testMatch: Glob[];
+        testPathIgnorePatterns: string[];
+        testRegex: string;
+        testResultsProcessor: Maybe<string>;
+        testRunner: Maybe<string>;
+        testURL: string;
+        timers: 'real' | 'fake';
+        transform: Maybe<{[key: string]: string}>;
+        transformIgnorePatterns: Glob[];
+        watchPathIgnorePatterns: string[];
+        useStderr: boolean;
+        verbose: Maybe<boolean>;
+        watch: boolean;
+        watchman: boolean;
+    }
+
+    interface InitialOptions {
+        automock?: boolean;
+        bail?: boolean;
+        browser?: boolean;
+        cache?: boolean;
+        cacheDirectory?: Path;
+        clearMocks?: boolean;
+        changedFilesWithAncestor?: boolean;
+        changedSince?: string;
+        collectCoverage?: boolean;
+        collectCoverageFrom?: Glob[];
+        collectCoverageOnlyFrom?: {[key: string]: boolean};
+        coverageDirectory?: string;
+        coveragePathIgnorePatterns?: string[];
+        coverageReporters?: string[];
+        coverageThreshold?: {global: {[key: string]: number}};
+        detectLeaks?: boolean;
+        detectOpenHandles?: boolean;
+        displayName?: string;
+        expand?: boolean;
+        filter?: Path;
+        findRelatedTests?: boolean;
+        forceCoverageMatch?: Glob[];
+        forceExit?: boolean;
+        json?: boolean;
+        globals?: ConfigGlobals;
+        globalSetup?: Maybe<string>;
+        globalTeardown?: Maybe<string>;
+        haste?: HasteConfig;
+        reporters?: Array<ReporterConfig | string>;
+        logHeapUsage?: boolean;
+        lastCommit?: boolean;
+        listTests?: boolean;
+        mapCoverage?: boolean;
+        moduleDirectories?: string[];
+        moduleFileExtensions?: string[];
+        moduleLoader?: Path;
+        moduleNameMapper?: {[key: string]: string};
+        modulePathIgnorePatterns?: string[];
+        modulePaths?: string[];
+        name?: string;
+        noStackTrace?: boolean;
+        notify?: boolean;
+        notifyMode?: string;
+        onlyChanged?: boolean;
+        outputFile?: Path;
+        passWithNoTests?: boolean;
+        preprocessorIgnorePatterns?: Glob[];
+        preset?: Maybe<string>;
+        projects?: Glob[];
+        replname?: Maybe<string>;
+        resetMocks?: boolean;
+        resetModules?: boolean;
+        resolver?: Maybe<Path>;
+        restoreMocks?: boolean;
+        rootDir?: Path;
+        roots?: Path[];
+        runner?: string;
+        runTestsByPath?: boolean;
+        scriptPreprocessor?: string;
+        setupFiles?: Path[];
+        setupTestFrameworkScriptFile?: Path;
+        silent?: boolean;
+        skipFilter?: boolean;
+        skipNodeResolution?: boolean;
+        snapshotSerializers?: Path[];
+        errorOnDeprecated?: boolean;
```

Comment:
Same as above.

Reply:
kk bump sounds good. Also I've been thinking about extracting those configs in separate file, this one ambient index is getting too big -> hard to maintain. WDYT ?

Reply:
Sounds good to me, and I think extracting all types mirrored from `jest` into separate files would be better, not just `Config.js`.

Reply:
I'll do that in separate PR ✌️. Thx for reviewing and your time @ikatyang  ! 🍻